docs/src/developers_guide/contributing_pytest_conversions.rst

```diff
@@ -50,6 +50,10 @@ Conversion Checklist
 #. Check for references to ``self.tmp_dir``. In pytest, ``tmp_path`` is used instead,
    and can be passed into functions as a fixture.
 #. Check for ``if __name__ == 'main'``. This is no longer needed with pytest.
+#. Remove the top-level import of :mod:`iris.tests` (usually ``import iris.tests as tests``).
+   Having followed the above steps, any remaining calls 
+   (e.g. :func:`iris.tests.get_data_path`) should be easily replacable with calls to
+   :mod:`iris.tests._shared_utils` (e.g. :func:`iris.tests._shared_utils.get_data_path`).
```

Comment:
Should this be its own bullet? Or would we be getting into a too-big list?

Reply:
7a64ea8ee67cc73aee2698f16579b180fc580184